modules/client/src/main/java/org/apache/ignite/internal/jdbc/JdbcStatement.java

```diff
@@ -131,7 +132,12 @@ protected void execute0(JdbcStatementType stmtType, String sql, List<Object> arg
         QueryExecuteRequest req = new QueryExecuteRequest(stmtType, schema, pageSize, maxRows, sql,
                 args == null ? ArrayUtils.OBJECT_EMPTY_ARRAY : args.toArray());
 
-        QueryExecuteResult res = conn.handler().queryAsync(req).join();
+        QueryExecuteResult res;
+        try {
+            res = conn.handler().queryAsync(req).join();
```

Comment:
Did you decide to apply this change together with cli changes? Is there a change to create a separate ticket?

Reply:
https://github.com/apache/ignite-3/pull/836